packages/storybook/src/stories/Alerts/Tooltip.stories.tsx

```diff
@@ -0,0 +1,64 @@
+  const noIcon = boolean('No Icon', false);
+  const iconList = { '': '', ...generateIconList() };
+  const icon = select("Icon", iconList, 'Information');
+  const tooltipPosition = select("Tooltip Position", { TopStart: 'top-start', TopCenter: 'top-center', TopEnd: 'top-end', BottomStart: 'bottom-start', BottomCenter: 'bottom-center', BottomEnd: 'bottom-end', LeftStart: 'left-start', LeftCenter: 'left-center', LeftEnd: 'left-end', RightStart: 'right-start', RightCenter: 'right-center', RightEnd: 'right-end' }, 'top-end')
+
+  return (
+    <Container>
+      <PageTitle
+        icon={'Home'}
+        title="Page with tooltips"
+      />
+      <Content>
+        The tooltip is a common graphical user interface (GUI) element in which, when <HoverSpan id='hoverSpanId'>hovering over</HoverSpan> a screen element or component, a text box displays information about that element, such as a description of a button's function, what an abbreviation stands for, or the exact absolute time stamp over a relative time ("… ago").
+      </Content>
+      <Content>
+        The tooltip is a common graphical user interface (GUI) element in which, when <HoverSpan id='hoverSpanId2'>hovering over</HoverSpan> a screen element or component, a text box displays information about that element, such as a description of a button's function, what an abbreviation stands for, or the exact absolute time stamp over a relative time ("… ago").
+      </Content>
+      <Content>
+        The tooltip is a common graphical user interface (GUI) element in which, when <HoverSpan id='hoverSpanId3'>hovering over</HoverSpan> a screen element or component, a text box displays information about that element, such as a description of a button's function, what an abbreviation stands for, or the exact absolute time stamp over a relative time ("… ago").
+      </Content>
+      <Content>
+        The tooltip is a common graphical user interface (GUI) element in which, when <HoverSpan id='hoverSpanId4'>hovering over</HoverSpan> a screen element or component, a text box displays information about that element, such as a description of a button's function, what an abbreviation stands for, or the exact absolute time stamp over a relative time ("… ago").
+      </Content>
+      <Content>
+        This tooltip <HoverSpan id='spanId5'>message</HoverSpan> position is fixed you can update it with the property tool position .
+      </Content>
+      <Tooltip tooltipFor='hoverSpanId' icon={noIcon ? undefined : icon} {...{ message }} />
+      <Tooltip tooltipFor='hoverSpanId2' icon={noIcon ? undefined : icon} {...{ message }} />
+      <Tooltip tooltipFor='hoverSpanId3' icon={noIcon ? undefined : icon} {...{ message }} />
+      <Tooltip tooltipFor='hoverSpanId4' icon={noIcon ? undefined : icon} {...{ message }} />
+      <Tooltip tooltipFor='spanId5' maxWidth='200px' icon={noIcon ? undefined : icon} {...{ type, message, tooltipPosition }} />
+    </Container>
```

Comment:
It's a bit unclear tht the last tooltip is the one that the knobs controls. 

Could you change the text in those triggers to help reflect which are static and which is the custom one?